deploy/Chart/templates/ucp/configmaps.yaml

```diff
@@ -36,7 +36,7 @@ data:
           resourceProviders:
             Applications.Core: "http://applications-rp.radius-system:5443"
             Applications.Dapr: "http://applications-rp.radius-system:5444"
-            Applications.Datastores: "http://applications-rp.radius-system:5444"
+            Applications.Datastores: "http://applications-rp.radius-system:5443"
```

Comment:
The same question applies to this too: What is the reason this has changed to 5443? For testing reasons?

Reply:
Same as above, we want to use one service for all RPs so we're updating the ports as we make the API changes